toolkit/imageconfigs/marketplace-gen1.json

```diff
@@ -22,6 +22,12 @@
                 {
                     "ID": "rootfs",
                     "Start": 9,
+                    "End": 999,
```

Comment:
`"End": 0` indicates the partition fills to end of disk. Instead of having the new partition be automatically filled to end of disk, which calculates out to be 500MB today, can we instead define the new 500MB partition explicitly and then let the rootfs partition be dynamically expanding to the end of the disk?

Reply:
I'd second that. We generally organize our disks as:
- efi
- boot
- misc-fixed-size

Then finish with one of two options with "End": "0":
- rootfs
of
-fixed_size_rootfs
-data

Feels odd to have a boot partition at the end, and especially dynamically sized. Cloutd-init also often tries to expand the rootfs to fill the VM, and it can only do that if its the last partition

Reply:
Thanks Chris and Daniel. I have moved this up, and put rootfs as the last partition